frontend/src/js/utils/locationutils.js

```diff
@@ -315,7 +315,7 @@ export const formatDeployments = ({ deploymentObject, pageState }, { defaults, t
   const { state: selectedState, showCreationDialog } = pageState.general;
   let params = new URLSearchParams();
   if (showCreationDialog) {
-    params.set('open', true);
+    params.set('open', 'true');
```

Comment:
```suggestion
    params.set('open', true);
```
`URLSearchParams` handles booleans